### DISCUSSION TITLE
Problemi con meros bulb mls120b

### DISCUSSION THREAD

**Discussion by barbatrukko** [indomus/forum#2728] [HUB: Home Assistant]
Salve,
premesso che ancora non mi è chiaro quale sia il plug-in migliore per gestire i dispositivi meross, sto avendo problemi con una lampadina.
Avevo già configurata una scena che si chiama ingresso_20 che mi porta la lampadina dell'ingresso al 20% di luminosità e luce calda.
Ho creato una automazione che quando la porta d'ingresso si apre e il sole è tramontato, attiva la scena ingresso MAX per 1 minuto e poi torna ad ingresso_20.
Ovviamente la scena ingresso MAX dovrebbe essere potenza al 100% e colore bianco freddo 6500K.
Questa configurazione io riesco a crearla dall'app sul cellulare, ma se poi mi copio i parametri tramite "strumenti per sviluppatore" e li metto in una scena da richiamare..... quel bianco luminoso non funziona..
Inoltre dall'interfaccia HA delle scene, se apro la configurazione della lampadina non riesco a fare molto (ma mi pare di ricordare che questo era un problema storico delle lampadine meross).

**Comment by KC-inDomus** [comment]
Usa [Meross LAN](https://indomus.it/guide/integrare-componenti-meross-con-home-assistant/)

**Reply by barbatrukko** [reply]
In passato però avevo notato che per le prese comandate (e con misurazione) il dato di potenza in tempo reale rilevato del MerosIOT era più accurato del Meros LAN, ovvero corrispondeva a quello letto dal'App di Meross.
